core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -200,26 +199,18 @@ public KafkaClusterTestKit build() throws Exception {
                     setupNodeDirectories(baseDirectory, node.metadataDirectory(),
                         node.logDataDirectories());
 
-                    // Just like above, we set a placeholder voter list here until we
-                    //find out what ports the controllers picked.
                     props.put(KafkaConfig$.MODULE$.ControllerQuorumVotersProp(), "");
-                    KafkaConfig config = new KafkaConfig(props, false,
-                        OptionConverters.toScala(Optional.empty()));
+                    KafkaConfig config = new KafkaConfig(props, false, OptionConverters.toScala(Optional.empty()));
 
                     String threadNamePrefix = String.format("broker%d_", node.id());
-                    LogContext logContext = new LogContext("[Broker id=" + node.id() + "] ");
-
-                    metaLogManager = new LocalLogManager(
-                        logContext,
-                        node.id(),
-                        sharedLogData,
-                        threadNamePrefix
-                    );
-                    metaLogManager.initialize();
-                    Kip500Broker broker = new Kip500Broker(
-                        config,
-                        nodes.brokerProperties(node.id()),
-                        metaLogManager,
+                    MetaProperties metaProperties = MetaProperties.apply(nodes.clusterId(),
+                            OptionConverters.toScala(Optional.empty()),
+                            OptionConverters.toScala(Optional.of(node.id())));
+                    TopicPartition metadataPartition = new TopicPartition(KafkaServer.metadataTopicName(), 0);
+                    KafkaRaftManager raftManager = new KafkaRaftManager(metaProperties, metadataPartition, config,
+                            Time.SYSTEM, new Metrics(), connectFutureManager.future);
```

Comment:
nit: use the existing `time` that's already in scope here

Reply:
Good catch. Thanks!